gui

```diff
@@ -172,6 +172,10 @@ runonce <<"EOF"
   refresh_all_pkgapp_status
 EOF
 
+runonce <<"EOF"
+  [ -a "$HOME/.local/share/applications/pi-apps-settings.desktop" ] && sed -i "s/logo.png/settings.png/g" "$HOME/.local/share/applications/pi-apps-settings.desktop"
```

Comment:
For the record, if the final command in a runonce exits with a non-zero exit code (indicating failure), the `runonce` function will note its failure and continue trying indefinitely.
A `true` command needs to be added before the EOF to prevent this from happening.

Reply:
This works for me:

```bash
[ -a "$HOME/.local/share/applications/pi-apps-settings.desktop" ] && sed -i "s/logo.png/settings.png/g" "$HOME/.local/share/applications/pi-apps-settings.desktop" || true
```